requirements/pip.txt

```diff
@@ -70,37 +70,38 @@ orjson==2.0.7  # pyup: ignore
 
 # Utils
 django-gravatar2==1.4.4
-pytz==2019.3
+pytz==2020.1
 Unipath==1.1
 django-kombu==0.9.4
-stripe==2.44.0
+stripe==2.47.0
 regex==2020.4.4
 
 # unicode-slugify==0.1.5 is not released on PyPI yet
 git+https://github.com/mozilla/unicode-slugify@b696c37#egg=unicode-slugify==0.1.5
 
-django-formtools==2.1
-django-crispy-forms==1.8.0
+django-formtools==2.2
+django-crispy-forms==1.9.0
 
-docker==4.1.0
+docker==4.2.0
 
 django-textclassifier==1.0
 # django-textclassifier doesn't have pinned versions
 # if there is an update they could break our code
-nltk==3.4.5
+nltk==3.5
 textblob==0.15.3
 
-django-annoying==0.10.5
-django-messages-extends==0.6.2 
+django-annoying==0.10.6
+django-messages-extends==0.6.2
 djangorestframework-jsonp==1.0.2
-django-taggit==1.1.0
+django-taggit==1.2.0
 dj-pagination==2.4.0
 
 # Version comparison stuff
-packaging==19.2
+packaging==20.3
 
-# Commenting stuff
-django-cors-middleware==1.4.0
+# django-cors-middleware==1.5.0 fails with
+# AttributeError: 'dict' object has no attribute 'has_header'
+django-cors-middleware==1.4.0  # pyup: ignore
```

Comment:
We should try to get this one fixed

Reply:
Yes. I think it will be better to tackle this in particular in a separate PR.